backend/app/assets/javascripts/spree/backend/taxonomy.js.coffee

```diff
@@ -116,6 +118,10 @@ root.setup_taxonomy_tree = (taxonomy_id) ->
                 node = m.o
                 new_parent = m.np
 
+                # wait for AJAX requests to finish
+                if $.active > 0
+                  return false
```

Comment:
Fixes an infinite loading error. If you try to drop an invalid taxon while another category is loading its child taxons, it'll load infinitely once the error handler has executed